src/plugins/vis_default_editor/public/components/controls/components/simple_number_list.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment, useCallback, useEffect, useMemo, useState } from 'react';
+import { isArray } from 'lodash';
+import { EuiButtonEmpty, EuiFlexItem, EuiFormRow, EuiSpacer, htmlIdGenerator } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EMPTY_STRING, getInitModelList, getRange, parse } from './number_list/utils';
+import { NumberRow, NumberRowModel } from './number_list/number_row';
+import { AggParamEditorProps } from '../../agg_param_props';
+
+const generateId = htmlIdGenerator();
+
+function SimpleNumberList({
```

Comment:
@maryia-lapata @DianaDerevyankina Could you please explain shortly what's the different between `SimpleNumberList` and `NumberList`, we use in other places in the editor (like percentiles)? Is the difference really large enough that it justifies having a separate component, or were there just minor things, that we could potentially solve adding a property to `NumberList`?

Reply:
The difference isn't big:
- `NumberList` creates next value as incremented the last one, `SimpleNumberList` creates just empty input;
- `NumberList`  saves an empty value as `undefined` to value, `SimpleNumberList` - as an empty string;
- `NumberList` can influence on the form validity and it uses `useValidation` hook as well as it validates each input, `SimpleNumberList` - not;
- there is no initial value in `SimpleNumberList`.

In case we decide to reuse `NumberList`, it's needed to make it more generic.  I think the component is already a bit complex (I wouldn't want to make it a `magic component`), and I'd leave a separate `SimpleNumberList` component.